lib/committee/request_unpacker.rb

```diff
@@ -87,5 +90,14 @@ def parse_json
         nil
       end
     end
+
+    def headers
+      env = @request.env
+      env.keys.grep(/HTTP_/).inject({}) do |headers, key|
+        headerized_key = key.gsub(/^HTTP_/, '').gsub(/_/, '-')
+        headers[headerized_key] = env[key]
+        headers
+      end
+    end
```

Comment:
I'm a tad bit concerned about the potential for different types of parameters to start colliding. For example, you could have a param called `Encoding` that appeared as both a header and as a form parameter, and Committee would have a hard time differentiating between the two.

What do you think about separating headers out into separate buckets? Maybe the best solution would be if `RequestUnpacker` returned a `[headers, params]` where each element could be validated separately, but a cheaper solution might be to just have a special `__committee_headers` key in the `params` hash for now that's extracted during validation. Thoughts?

Reply:
As you pointed out, I think header and parameter are orthogonal too.
In current PR implementation, collision is occured and this is not good for us.
So separating headers out into separate buckets is good idea.

I think the best solution is separeting params and headers on RequestUnpacker#call too.

The following is idea for implementation.

* `Committee::Drivers::OpenAPI2::Link` has new `header_schema` attribute that is `JsonSchema::Schema` Object to validate header.
  * `Committee::Drivers::HyperSchema` does't have `header_schema`. This attribute only be in OpenAPI.
* `RequestUnpacker` returns array, params and headers.
* `RequestValidator` validates params and headers separately. params are validated with Link#schema, headers are validated with Link#header_schema.

If the above solution looks good, I'll implement it.

Reply:
Thanks for thinking it through so thoroughly! Yes, that sounds good to me.